pkg/settlement/swap/metrics.go

```diff
@@ -32,6 +34,18 @@ func newMetrics() metrics {
 			Name:      "total_sent",
 			Help:      "Amount of tokens sent to peers (costs paid by the node)",
 		}),
+		ChequesReceived: prometheus.NewCounter(prometheus.CounterOpts{
+			Namespace: m.Namespace,
+			Subsystem: subsystem,
+			Name:      "cheques_received",
+			Help:      "Amount of cheques received from peers",
+		}),
+		ChequesSent: prometheus.NewCounter(prometheus.CounterOpts{
+			Namespace: m.Namespace,
+			Subsystem: subsystem,
+			Name:      "cheques_sent",
+			Help:      "Amount of cheques sent to peer",
```

Comment:
to peers

Reply:
fixed. I also changed amount of to number of, as it was also that way in the existing metrics that track number of cheques.